src/components/inc/Mywork.js

```diff
@@ -0,0 +1,112 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import Slider1 from '../images/fullstack.png';
+import Slider2 from '../images/portfolio.jpg';
+import Slider3 from '../images/app.jpg';
+import { tab } from '@testing-library/user-event/dist/tab';
+
+function Mywork() {
+    return (
+        <div className='container mb-5'>
+            <div className='row '>
+                <div className='col-md-12 mb-4 text-center'>
+                    <h3 className='main-heading mt-4'>Our Works</h3>
+                    <div className='underline mx-auto'></div>
+                </div>
+            </div>
+            <div className='border rounded'>
+                <nav className='justify-content-center'>
+                    <div className='nav nav-tabs' id='nav-tab' role='tablist'>
+                        <button className='nav-link active px-3 fw-bold' id='nav-home-tab' data-bs-toggle='tab' data-bs-target="#nav-home" type='button' role={tab} aria-controls='nav-home' aria-selected='true'>
+                            Web Applications
+                        </button>
+
+                        <button className='nav-link px-3 fw-bold' id='nav-profile-tab' data-bs-toggle='tab' data-bs-target="#nav-profile" type='button' role={tab} aria-controls='nav-profile' aria-selected='false'>
+                            Full Stack Applications
+                        </button>
+
+                        <button className='nav-link px-3 fw-bold' id='nav-contact-tab' data-bs-toggle='tab' data-bs-target="#nav-contact" type='button' role={tab} aria-controls='nav-contact' aria-selected='false'>
+                            Mobile Applications
+                        </button>
+                    </div>
+                </nav>
+                <div className='tab-content' id='nav-tabContent'>
```

Comment:
Always loop these components.... we cannot keep on writing all the works here that takes lot of lines of code.